### ISSUE TITLE
[Accessibility - multiple pages] Navigation order on left panel 

### ISSUE THREAD

**Issue by lpalmaro** [Polymer/docs#1140]
Within the panel, when using VoiceOver with Mac, it appears that the order of navigation is a little odd. It should be presented in the same visual order from left to right, e.g. "local dom" THEN "keyboard arrow down image". But it's the opposite. The keyboard arrow down option actually comes first for each item, so it's confusing for a user to know what that keyboard arrow down is associated with. We should consider adding a more descriptive label to each arrow, e.g. "local dom keyboard arrow down image" so it's clearer.
